R/getRunStatus.R

```diff
@@ -200,12 +194,15 @@ getRunStatus <- function(mydir = dir(), sort = "nf", user = NULL) {
       if (exists("stats")) {
         if (any(grepl("GAMSEnd", names(stats)))) {
           out[i, "Runtime"] <- format(round(stats[["timeGAMSEnd"]] - stats[["timeGAMSStart"]], 1))
```

Comment:
when run is "Normal completion" but end time has not been written, print the time since start time instead of NA